src/index.html

```diff
@@ -19,9 +19,400 @@
       rel="stylesheet"
       href="styles/index.scss"
     />
+
+    <link
+      rel="preconnect"
+      href="https://fonts.googleapis.com"
+    />
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+      crossorigin="anonymous"
+    />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap"
+      rel="stylesheet"
+    />
   </head>
 
   <body>
-    <h1>Catalog</h1>
+    <header class="header">
+      <a
+        href="#"
+        class="header__logo"
+      >
+        <img
+          src="images/logo.png"
+          alt="Moyo Company Logo"
+          class="header__logo-img"
+        />
+      </a>
+
+      <nav class="nav">
+        <ul class="nav__list">
+          <li class="nav__item">
+            <a
+              href="#"
+              class="nav__link is-active"
+            >
+              Apple
+            </a>
+          </li>
+          <li class="nav__item">
+            <a
+              href="#"
+              class="nav__link nav__link--margin-left"
+            >
+              Samsung
+            </a>
+          </li>
+          <li class="nav__item">
+            <a
+              href="#"
+              class="nav__link nav__link--margin-left"
+            >
+              Smartphones
+            </a>
+          </li>
+          <li class="nav__item">
+            <a
+              href="#"
+              class="nav__link nav__link--margin-left"
+              data-qa="nav-hover"
+            >
+              Laptops & Computers
+            </a>
+          </li>
+          <li class="nav__item">
+            <a
+              href="#"
+              class="nav__link nav__link--margin-left"
+            >
+              Gadgets
+            </a>
+          </li>
+          <li class="nav__item">
+            <a
+              href="#"
+              class="nav__link nav__link--margin-left"
+            >
+              Tablets
+            </a>
+          </li>
+          <li class="nav__item">
+            <a
+              href="#"
+              class="nav__link nav__link--margin-left"
+            >
+              Photo
+            </a>
+          </li>
+          <li class="nav__item">
+            <a
+              href="#"
+              class="nav__link nav__link--margin-left"
+            >
+              Video
+            </a>
+          </li>
+        </ul>
+      </nav>
+    </header>
+
+    <main class="catalog">
+      <div
+        class="product-card"
+        data-qa="card"
+      >
+        <img
+          class="product-card__image"
+          src="images/imac.jpeg"
+          alt="iMac Image"
```

Comment:
Check [checklist](https://mate-academy.github.io/layout_task-guideline/html-css-code-style-rules)
<img width="1138" alt="Знімок екрана 2024-10-24 о 17 43 43" src="https://github.com/user-attachments/assets/191853bb-d17b-48a9-afe3-3dfcdb6ed351">

```suggestion
          alt="......."
```